suites/apis/requestorTest.js

```diff
@@ -1,84 +1,81 @@
+  users, fence,
 }) => {
-  home.do.goToHomepage();
-  login.complete.login(users.user0);
-
   // Check that the user does not have access to policy 'requestor_integration_test'.
   let userInfo = await fence.do.getUserInfo(users.user0.accessToken);
-  expect(userInfo.data.authz).to.have.property('/requestor-integration-test');
-  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.not.include({ method: 'read', service: 'integration' });
+  expect(userInfo.data.authz).to.not.have.property('/programs/requestor-integration-test');
 
+  const { accessTokenHeader } = users.mainAcct;
   // Create a request for a policy that is present in Arborist
-  const createResponse = await requestorTasks.createRequestFromPolicyID(users.user0, 'requestor_integration_test');
+  const createResponse = await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test');
   expect(createResponse).to.have.property('status_code', 201);
 
   // Check if the access is created -- It shouldn't since it is not signed yet
   userInfo = await fence.do.getUserInfo(users.user0.accessToken);
-  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.not.include({ method: 'read', service: 'integration' });
+  expect(userInfo.data.authz).to.not.have.property('/programs/requestor-integration-test');
 
   // Create a PUT request to sign the previous request using an authenticated user
   await requestorTasks.signedRequest(createResponse.request_id);
 
+  const user0AccessToken = users.user0.accessToken;
   // Verify if access is given to the user -- There should be
-  userInfo = await fence.do.getUserInfo(users.user0.accessToken);
-  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.include({ method: 'read', service: 'integration' });
+  userInfo = await fence.do.getUserInfo(user0AccessToken);
+  expect(userInfo.data.authz).to.have.property('/programs/requestor-integration-test');
+  expect(userInfo.data.authz['/programs/requestor-integration-test']).to.deep.to.include({ method: 'access', service: 'jupyterhub' });
 
   // Create a request to 'revoke' a policy that is present in Arborist
-  const revokeResponse = await requestorTasks.createRequestFromPolicyID(users.user0, 'requestor_integration_test', true);
+  const revokeResponse = await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test', true);
 
   // Check if the access is revoked -- It shouldn't since it is not signed yet
-  userInfo = await fence.do.getUserInfo(users.user0.accessToken);
-  expect(userInfo.data.authz['/requestor-integration-test']).to.deep.to.include({ method: 'read', service: 'integration' });
+  userInfo = await fence.do.getUserInfo(user0AccessToken);
+  expect(userInfo.data.authz['/programs/requestor-integration-test']).to.deep.to.include({ method: 'access', service: 'jupyterhub' });
```

Comment:
i think you might want to add this line here too, because if the access was revoked before being signed, the resource would not be there:
`expect(userInfo.data.authz).to.have.property('/programs/requestor-integration-test');`